homeassistant/components/smarttub/sensor.py

```diff
@@ -6,25 +6,52 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_DURATION = "duration"
+ATTR_LAST_UPDATED = "last_updated"
```

Comment:
Cruft ?

Reply:
Yes, these belong in https://github.com/home-assistant/core/pull/46868, will fix